Changelogs for Digital Twin Manager Plugins 3.2.1 (#3697)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1022-6-0-use-existing-assign-devices-component-from-subassets-in-add-asset.md

---
date: ""

> title: "Use existing Assign devices component from Subassets in Add Asset"

Reviewer: ```suggestion
title: Aligned Add asset dialog with device assignment from Subassets view
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2330"
version: "1022.6.0"
---

> Previously, in the Add Asset plugin, a toggle button was used to view the child devices during device assignment. Now, that toggle has been removed, and the button is directly displayed beside the parent device on the right side of the grid.

Reviewer: I don´t think that it is relevant for the user that this is the add asset plugin.
```suggestion
Previously, when assigning devices in the Add asset dialog, child devices could be viewed by using a toggle button. Now, that toggle has been removed, and the button is directly displayed beside the parent device on the right side of the grid.
```

Reviewer: Also, shouldn´t you mention that this is already used in other views as the title seems to say?

> Previously, when assigning devices in the Add asset dialog, child devices could be viewed by using a toggle button. Now, that toggle has been removed, and the button is directly displayed beside the parent device on the right side of the grid similar to the device assignment from Subassets view.

Reviewer: ```suggestion
Previously, when assigning devices in the Add asset dialog, child devices could be viewed by using a toggle button. Now, that toggle has been removed, and the button is directly displayed beside the parent device on the right side of the grid, similar to the device assignment in the Subassets view.
```
